kimchi/src/circuits/lookup/index.rs

```diff
@@ -60,7 +60,7 @@ struct LookupSelectorsSerdeAs<F: FftField> {
     #[serde_as(as = "Option<o1_utils::serialization::SerdeAs>")]
     pub lookup_gate: Option<E<F, D<F>>>,
     #[serde_as(as = "Option<o1_utils::serialization::SerdeAs>")]
-    pub range_check_gate: Option<E<F, D<F>>>,
+    pub range_check: Option<E<F, D<F>>>,
```

Comment:
Same comment as above.  To keep the diff smaller and keep naming consistent, there's no reason to rename these variables.

Reply:
I believed a change could be nice to be uniform with respect to the Xor pattern

Reply:
I prefer it follow the mnemonics of `lookup_gate` et al also it's not related to this PR, so should not be done here.